src/tools/wasm-split.cpp

```diff
@@ -443,14 +443,23 @@ void Instrumenter::addProfileExport() {
   // TODO: export the memory if it is not already exported.
 }
 
+uint64_t hashFile(const std::string& filename) {
+  auto contents(read_file<std::vector<char>>(filename, Flags::Binary));
+  size_t digest = 0;
+  // Don't use `hash` or `rehash` - they aren't deterministic between executions
+  for (char c : contents) {
+    hash_combine(digest, c);
+  }
+  return uint64_t(digest);
```

Comment:
`digest` is `size_t`, then why do we need `uint64_t` to store a module hash?

Reply:
Using uint64_t makes the profile parsing simpler because we can unconditionally use 64 bits in the profile rather than having to determine whether to use 32 or 64 bits depending on whether we are running on a 32-bit platform.

Reply:
But it seems we are using only 32 bits, then can't we use 32 bits unconditionally?

Reply:
Oh, sorry, I somehow assumed `size_t`'s size is the same as `unsigned int`... it seems to be implementation-defined. Nevermind.